### DISCUSSION TITLE
It's flashing, is there anyway i can do something about it?

### DISCUSSION THREAD

**Discussion by ogios** [charmbracelet/bubbletea#994] [Q&A] [answered]
https://github.com/charmbracelet/bubbletea/assets/96933655/5c94ed3b-5265-4f3d-a5d1-ff5c3acbd50c

**Comment by matta** [comment, marked as answer]
See issue #1032. Seems like it is a known issue and is being worked on.

**Reply by meowgorithm** [reply]
That's correct! It's under heavy development at the moment and is one of Bubble Tea's current top priorities.
